caampr-backend/src/test/java/com/jelistan/caampr/lambda/adaptor/GearListRequestAdaptorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.jelistan.caampr.lambda.adaptor;
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.jelistan.caampr.lambda.model.GearListRequest;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class GearListRequestAdaptorTest {
+    @Mock
+    private APIGatewayProxyRequestEvent requestEvent;
+
+    private GearListRequestAdaptor adaptor;
```

Comment:
For this project, the class under test should be named `unit`.  This helps to identify things